src/BlockInfo.cpp

```diff
@@ -584,6 +584,27 @@ void cBlockInfo::Initialize(cBlockInfoArray & a_Info)
 	a_Info[E_BLOCK_STONE               ].m_CanBeTerraformed = true;
 
 
+	// Block heights:
+	a_Info[E_BLOCK_BED                 ].m_BlockHeight = 9.0 / 16.0;
+	a_Info[E_BLOCK_CAKE                ].m_BlockHeight = 8.0 / 16.0;
```

Comment:
Now it's "consistent" and looks awful. I added comments as reminders to where the magic values supposedly come from.